app/view/main/Main.js

```diff
@@ -44,5 +45,7 @@ Ext.define('CpsiMapview.view.main.Main', {
         }
     }, {
         xtype: 'cmv_header'
+    }, {
+        xtype: 'cmv_add_wms_window'
```

Comment:
How about instantiating the `cmv_add_wms_window` in the handler of the button instead of hanging it in the main view?